doc/book/box/limitations.rst

```diff
@@ -49,6 +49,15 @@ Limitations
     such as :ref:`Number of bytes in a tuple <limitations_bytes_in_tuple>`,
     not by the index support.
 
+.. _limitations_fields_in_tuple_multikey_index:
+
+**Number of elements in array fields in a space with a multikey index**
+
+    In a Tarantool space that has :ref:`multikey indexes <box_space-path_multikey>` on array fields,
```

Comment:
Maybe just `on fields`?
```suggestion
    In a Tarantool space that has :ref:`multikey indexes <box_space-path_multikey>` on fields,
```
Because in example below we have a map field on the first level and in space format we could have map `format = {{name=..., type='map'}}`